pkg/acpi/ibft.go

```diff
@@ -58,7 +58,10 @@ type acpiIBFTHeader struct {
 	Reserved   [24]byte `offset:"24" desc:"Reserved"`
 }
 
-var rawIBTFHeader = "IBFT\x00\x08\x00\x001\x00ACPIXXACPISUCK1234567890abcdef01234567"
+var (
+	rawIBTFHeader = "IBFT\x00\x08\x00\x001\x00ACPIXXACPISUCK1234567890abcdef01234567"
+	_             = Tabler(&IBFT{})
```

Comment:
Why do we discard this tabler?

Reply:
this is a standard way to assert that a type implements an interface. So in this case we're forcing a cast, but we don't need the variable, so we use the blank.